php-nginx/build-scripts/build_php70.sh

```diff
@@ -49,6 +49,9 @@ rm mailparse.tar.gz
 # rm apcu.tar.gz
 git clone https://github.com/krakjoe/apcu.git ${PHP_SRC}/ext/apcu
 
+# APC compatibility layer
+git clone https://github.com/krakjoe/apcu-bc.git ${PHP_SRC}/ext/apc
```

Comment:
https://pecl.php.net/package/apcu_bc
It seems like apcu_bc is available on pecl. Is it possible to download a stable version from pecl?